wiki/Guides/Videos_from_YouTube/en.md

```diff
@@ -0,0 +1,29 @@
+---
+tags:
+  - how to download videos from YouTube
+  - how to use YouTube videos
+---
+
+# Videos from YouTube
+
+[YouTube](https://youtube.com) is a very convenient resource. If a song has a music video, it will most likely be on YouTube, and probably compressed at that golden point where the video looks decent but will not be a pain to download.
```

Comment:
```suggestion
[YouTube](https://youtube.com "YouTube") is a very useful resource for creators looking to add background videos to their [beatmaps](/wiki/Beatmaps): if a song has a music video, it will most likely be on YouTube. (And it will probably be compressed at that golden point where the video looks decent but will not be a pain to download.)

There are many ways to obtain videos from YouTube onto yoru device. Below, we've listed a few of the most popular ways of obtaining such videos:
```
Maybe try this?